### PR TITLE
ci: add coverage step as part of CI pipeline

### PR DESCRIPTION
**Pull request by mariajgrimaldi**
**Description:**
This PR adds coverage step to CI GitHub pipeline.

**Dependencies:**
None

**Merge deadline:** 
ASAP

**Installation instructions:** 
None. New GitHub pipeline step.

**Testing instructions:**
None. New GitHub pipeline step. Check out the org coverage page for openedx-events: [here](https://app.codecov.io/gh/openedx/openedx-events/pull/111)

**Reviewers:**
- [x] @felipemontoya 

**Merge checklist:**
- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Version bumped
- [ ] Changelog record added
- [ ] Documentation updated (not only docstrings)
- [ ] Commits are squashed

**Post merge:**
- [ ] Create a tag
- [ ] Check new version is pushed to PyPI after tag-triggered build is 
      finished.
- [ ] Delete working branch (if not needed anymore)

**Author concerns:** 
None

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @mariajgrimaldi!

As a core committer in this repo, you can merge this once the pull request is approved per the [core committer reviewer requirements](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1529675973/Open+edX+Core+Committers#Reviews-and-Ownership) and according to the agreement with your edX Champion.

File: .github/workflows/ci.yml
Comment (felipemontoya) [inline .github/workflows/ci.yml:43]: Do we want https://github.com/openedx/DoneXBlock/blob/master/.github/workflows/ci.yml#L43?

**Review by felipemontoya — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@mariajgrimaldi 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/openedx-events&entry.867055334=https://github.com/openedx/openedx-events/pull/111&entry.1484655318=https://github.com/mariajgrimaldi&entry.752974735=2022-09-12+18:57&entry.1917517419=2022-09-13+18:12&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**
